epub.go

```diff
@@ -75,6 +75,7 @@ img {
 	defaultCoverImgFormat     = "cover%s"
 	defaultCoverXhtmlFilename = "cover.xhtml"
 	defaultEpubLang           = "en"
+	defaultPpd                = "ltr"
```

Comment:
I wonder if "default" would be a better default value based on the spec. What do you think?:

> When the default value is specified, the Author is expressing no preference and the Reading System may chose the rendering direction. This value must be assumed when the attribute is not specified.
http://www.idpf.org/epub/301/spec/epub-publications.html#sec-spine-elem